sla.md

```diff
@@ -0,0 +1,29 @@
+---
+---
+
+# Wobscale's Service Level Agreement
+
+Wobscale provides a nine fives [SLA](https://en.wikipedia.org/wiki/Service-level_agreement) for all services and hardware we're responsible for.
+
+## What it is
+
+The Wobscale nine-fives SLA promises that a given service will be available
+at least 55.5555555% of each month.
+
+In reality, this is all just a pun on the more common [five nines](https://en.wikipedia.org/wiki/Five_nines), and we will strive to do far better than we promise.
+
+That being said, we are running this as a hobby. We'll try our darndest to keep
+things running well, but we don't have 24/7 oncall nor a stack of spare
+equipment for every situation.
+
+## Breaking the SLA
+
+If the SLA is broken, we will refund you the full cost of the service for the month in question, plus an additional 55 cents (plus or minus taxes).
+
+Terms and conditions may apply and this portion of the agreement may change without notice, even retroactively.
+
+## What it covers
```

Comment:
It probably shouldn't cover pre-announced downtime (e.g. our upcoming switch to Wob Net 2 whenever I finally get that shit working)

Reply:
I think it's such a generous number it doesn't matter much either way.

If we do manage to have planned downtime long enough to even approach this, we've done something quite wrong and should be accountable regardless.